tasks/fix-cat1.yml

```diff
@@ -124,7 +124,6 @@
 
 
 - name: |
-
```

Comment:
I understand the reason this newline was added, but I'm torn as to whether to include it.  W/o it, the two following lines don't line up on execution, but it looks messy in the playbook...  FWIW, I've been not including this newline in task blocks I've written, consistent w/ your change here.

Reply:
Yes I think I remember this discussion now. I removed it because it made it easier for the doc generator to parse. I'm one to lean towards not including it since I don't typically watch task output in the console.